schemas/0.11.0/fmu_results.json

```diff
@@ -1013,6 +1013,24 @@
       "title": "DomainReference",
       "type": "string"
     },
+    "Element": {
+      "description": "The ``fmu.element`` block identifies data objects representing the same entity\nwithin a case, i.e. both realizations and aggregations of an element will share\nthe same unique identifier.",
+      "properties": {
+        "id": {
```

Comment:
`uuid` or `id`? 🤔

Reply:
I was not so sure here actually, but I think I lean towards `uuid` when I think about it one more time. That will be more descriptive of what it actually is. And I see `uuid` is the field name we mostly use elsewhere in the datamodel.. although I see we are not consistent e.g. the `fmu.aggregation` uses `id`.